tests/Naming/SmartUniqidNamerTest.php

```diff
@@ -18,11 +18,11 @@ public function fileDataProvider(): array
             'special char' => ['yezz!.png', '/yezz-[[:xdigit:]]{22}\.png/'],
             'long basename' => [\str_repeat('a', 256).'.txt', '/a{228}-[[:xdigit:]]{22}\.txt/'],
             'long extension' => ['a.'.\str_repeat('a', 256), '/a-[[:xdigit:]]{22}\.a{230}/'],
-            'long basename and extension' => [\str_repeat('a', 256).'.txt'.\str_repeat('a', 256),
-                                              '/a{228}-[[:xdigit:]]{22}\.txt/', ],
-            'double extension' => ['lala.png.jpg', '/lala\.png-[[:xdigit:]]{22}\.jpg/'],
+            'long basename and extension' => [\str_repeat('a', 256).'.txt'.\str_repeat('a', 256), '/a{228}-[[:xdigit:]]{22}\.txt/'],
```

Comment:
Please do not alter unrelated rows

Reply:
@garak thanks for your feedback. I've updated my commit.